e2e-tests/smart-update/run

```diff
@@ -6,69 +6,74 @@ set -o xtrace
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 
+function check_pod_update() {
+	local pod_name=$1
+	local img=""
+
+	echo "wait for ${pod_name} update"
+	set +o xtrace
+	retry=0
+	while true; do
+		sleep 1
+		echo -n .
+		let retry+=1
+		img=$(kubectl get pod/${pod_name} -o jsonpath='{.spec.containers[0].image}')
+		if [ "${img}" == "${IMAGE_MONGOD_TO_UPDATE}" ]; then
+			echo "OK: Image ${img} was updated for pod ${pod_name}!"
+			break
+		elif [ ${retry} -ge 60 ]; then
+			echo "ERROR: Image was not updated for pod ${pod_name}! Image is ${img}, but should be ${IMAGE_MONGOD_TO_UPDATE}."
+			exit 1
+		fi
+	done
+	set -o xtrace
+	wait_pod "${pod_name}"
+}
+
 cluster="smart-update"
-create_namespace $namespace
+create_namespace ${namespace}
 deploy_operator
 
-kubectl_bin apply -f $conf_dir/secrets.yml -f $conf_dir/client.yml
+kubectl_bin apply -f ${conf_dir}/secrets.yml -f ${conf_dir}/client.yml
 
```

Comment:
What is the difference between this two variants? I tested it and had  the same results.

Reply:
there's no difference, just wanted to not have a mix in the same script because it looks ugly, so put all variables with braces